masterlist.yaml

```diff
@@ -5969,54 +5969,28 @@ plugins:
       - Delev
       - Graphics
       - Invent
-      - NpcFaces
-      - NPC.Race
       - Relev
-    dirty:
-      # v4.4
-      - <<: *quickClean
-        crc: 0xF789959A
-        util: '[FNVEdit v4.0.2f](https://www.nexusmods.com/newvegas/mods/34703)'
-        itm: 94
-        udr: 73
-  - name: 'New Vegas Redesigned 3.esp'
-    url: [ 'https://www.nexusmods.com/newvegas/mods/56312/' ]
-    inc:
-      - 'New Vegas Redesigned.esm'
-      - 'New Vegas Redesigned II.esm'
-      - 'TTW Redesigned.esm'
-    tag:
-      - Actors.ACBS
-      - Actors.AIData
-      - Actors.AIPackages
-      - Delev
-      - Graphics
-      - Invent
-      - NpcFaces
+      - NPC.Eyes
+      - NPC.FaceGen
+      - NPC.Hair
       - NPC.Race
-      - Relev
-    dirty:
-      # v4.4
-      - <<: *quickClean
-        crc: 0xA49C36C8
-        util: '[FNVEdit v4.0.2f](https://www.nexusmods.com/newvegas/mods/34703)'
-        itm: 1001
-        udr: 74
```

Comment:
I would keep the cleaning data as well, just in case anyone installs the old version.
But I'm flexible on that, if it's pretty well known that this version shouldn't be used.

Does either version provide version info in the plugin file header?
If the new one does, than there could be a case for removing the cleaning data.
As we could inform them of the newer version.

Reply:
I'll check rn. And good idea.

Reply:
Okay, so both versions on the Header say "1.340000". That is what you're asking for, right?

Reply:
> Okay, so both versions on the Header say "1.340000". That is what you're asking for, right?

It should be under `SNAM - Description`

![image](https://github.com/loot/falloutnv/assets/1944639/24fcf126-01be-4576-83c7-1555a4e306bb)

Reply:
In that case, no. Nothing in SNAM.

Reply:
Well you could check the new version files and see if they are clean.
If they are clean then the CRC is reliable, and we could use that for an update message.
And then there would be no need to keep the cleaning data for the old version.

Reply:
Fair fair! I shall do it!

Reply:
clean:
      - crc: 0x972A47BB
        util: 'FNVEdit v4.1.5 EXPERIMENTAL'

As expected, it is indeed clean.